zerver/migrations/0461_alter_realm_default_code_block_language.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.2 on 2023-07-11 14:04
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def set_default_code_block_language_to_empty_string(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    for realm in Realm.objects.filter(default_code_block_language=None):
+        realm.default_code_block_language = ""
+        realm.save(update_fields=["default_code_block_language"])
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0460_backfill_realmauditlog_extradata_to_json_field"),
+    ]
+
+    operations = [
+        # Change default value from None to empty string for new realms
+        migrations.AlterField(
+            model_name="realm",
+            name="default_code_block_language",
+            field=models.TextField(null=True, default=""),
+        ),
```

Comment:
I split the change on the realm field into three steps so that when the migration was run on servers that supported creating new realms (like Zulip Cloud) that the default was first set to `""`.

Then the migration runs to update existing realms that have `None` set for the field to have the empty string set, and then finally removes the `null=True` on the field.

If that's not necessary, then the first `migrations.AlterField` change can be removed.